visidata/guides/DirSheet.md

```diff
@@ -0,0 +1,35 @@
+---
+sheettype: DirSheet
+---
+# Directory Sheet
+
+The **DirSheet** is a display of files and folders in the *source* directory.
+
+To load a **DirSheet**:
+
+- provide VisiData a directory as input (e.g. [:code]vd sample_data/[/])
```

Comment:
- don't have to mention "VisiData" in guides.  "provide a directory as a source on the CLI"